public/docs/_examples/toh-1/ts/app/app.component.ts

```diff
@@ -13,10 +13,14 @@ export class Hero {
   template:`
     <h1>{{title}}</h1>
     <h2>{{hero.name}} details!</h2>
-    <div><label>id: </label>{{hero.id}}</div>
+    <dl>
+      <dt>id:</dt>
+      <dd>{{hero.id}}</dd>
+    </dl>
     <div>
-      <label>name: </label>
-      <input [(ngModel)]="hero.name" placeholder="name">
+      <label>name: 
```

Comment:
Nah that one is outdated. Did that, then life happened, with some changes and edits. I just created a new one at https://github.com/angular/angular.io/pull/1625

Agree to a certain extent but what we don't want are devs copying and pasting it from **TOH** because they think *that is what is needed to make Angular work* 

I think most devs looking here will be mostly concerned with getting their first app off the ground and we should not confuse them with extra code. So in the **TOH** examples I think we should look at the big wins that require little to no extra code. The rest will come later.